composer.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "paysafegroup/paysafe_sdk_php",
+    "description": "Paysafe PHP SDK",
+    "keywords": ["paysafe","php", "netbanx", "optimalpayments", "sdk"],
+    "license": "MIT",
+    "require": {
+        "php": ">=5.3.0"
+    },
+    "autoload": {
```

Comment:
Cool. I noticed on the old version of the SDK (https://github.com/OptimalPayments/PHP_SDK) they defined a composer.json w/ a `files` autoload element that brought in the custom autoloader. Personally, I think your approach, leveraging PSR-4, is an improvement.